src/platform/cannonlake/include/platform/mem_map.h

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Is there any need for this new header ? Can we just use memory.h.

Reply:
@lgirdwood 
We can not use directly memory.h with platform_memmap_init function declaration, because it is included in assembler and linker sources.

Reply:
@bkokoszx  wee need to keep all our platform memory definition and declarations in memory.h. We can protect C declarations using
```
#ifdef ASSEMBLER
```
and also declare a
```
-DLINKER
```
during linker script pre-processing.

Reply:
@lgirdwood thanks, updated